packages/shared/src/hooks/useRegistration.ts

```diff
@@ -92,11 +92,11 @@ const useRegistration = ({
 
   const onValidateRegistration = async (values: RegistrationParameters) => {
     const { nodes, action } = registration.ui;
-    const csrfToken = getNodeByKey('csrf_token', nodes);
     const postData: RegistrationParameters = {
       ...values,
       method: values.method || 'password',
-      csrf_token: csrfToken.attributes.value,
+      csrf_token: getNodeValue('csrf_token', nodes),
+      'traits.image': values['traits.image'] || fallbackImages.avatar,
```

Comment:
I think I removed this, because it was always pre-overwritting the social sign-in.
We only need to set this on the actual registration call (the last one)